app/services/get_referees_to_chase.rb

```diff
@@ -0,0 +1,9 @@
+class GetRefereesToChase
+  def perform
+    ApplicationReference
+    .joins(:application_form)
+    .feedback_requested
+    .where(['application_forms.submitted_at < ?', Time.zone.now - 5.days])
+    .select { |reference| reference.chasers_sent.reference_request.blank? }
```

Comment:
```suggestion
    .where.not(id: ChaserSent.reference_request.select(:chased_id))
```

This will generate one query for this call:

```
[3] pry(main)> puts GetRefereesToChase.new.perform.to_sql
SELECT "references".* FROM "references" INNER JOIN "application_forms" ON "application_forms"."id" = "references"."application_form_id" WHERE "references"."feedback_status" = 'feedback_requested' AND (application_forms.submitted_at < '2020-01-30 15:07:56.592003') AND "references"."id" NOT IN (SELECT "chasers_sent"."chased_id" FROM "chasers_sent" WHERE "chasers_sent"."chaser_type" = 'reference_request')
```